llvm/utils/lit/lit/reports.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, output_file):
         self.output_file = output_file
         # Set by the option parser later.
         self.use_unique_output_file_name = False
+        self.skipped_codes = {lit.Test.EXCLUDED,
+                              lit.Test.SKIPPED, lit.Test.UNSUPPORTED}
```

Comment:
This made it's way over from patch #1 (https://reviews.llvm.org/D143516). However, since it's not required for my changes, I will split it into another PR as you are suggesting.